src/views/catalog/wizard/Wizard.tsx

```diff
@@ -1,14 +1,26 @@
 import * as React from 'react';
 import { Trans } from 'react-i18next';
+import { useParams } from 'react-router-dom';
 
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { HorizontalNav } from '@openshift-console/dynamic-plugin-sdk';
 import { Stack, StackItem, Title } from '@patternfly/react-core';
 
+import { useWizardVMContext } from '../utils/WizardVMContext';
+
+import { WizardEmptyState } from './components/WizardEmptyState';
+import { WizardFooter } from './components/WizardFooter';
 import { wizardNavPages } from './tabs';
 
+import './Wizard.scss';
+
 const Wizard: React.FC = () => {
   const { t } = useKubevirtTranslation();
+  const { ns } = useParams<{ ns: string }>();
+  const { vm } = useWizardVMContext();
+
+  if (!vm) return <WizardEmptyState namespace={ns} />;
```

Comment:
Shouldn't it be `<NoVMInContexts ... >`  not `<WIzardEmptyState ... >` ?

Reply:
EmptyState is a convention for components with missing props/state.
Naming a component starting with `<No` is not a great idea IMO